source/frontend/src/features/properties/map/acquisition/common/update/acquistionOwners/UpdateAcquisitionOwnersSubForm.tsx

```diff
@@ -0,0 +1,113 @@
+import { FieldArray, useFormikContext } from 'formik';
+import { useState } from 'react';
+import { Container, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+import { AcquisitionFormModal } from '../../../modals/AcquisitionFormModal';
+import { AcquisitionOwnerFormModel, WithAcquisitionOwners } from '../../models';
+
+interface IUpdateAcquisitionOwnersSubFormProps {}
+
+export const UpdateAcquisitionOwnersSubForm: React.FunctionComponent<
+  React.PropsWithChildren<IUpdateAcquisitionOwnersSubFormProps>
+> = () => {
+  const { values } = useFormikContext<WithAcquisitionOwners>();
+  const [removeIndex, setRemoveIndex] = useState<number>(-1);
+  const [showRemoveModal, setShowRemoveModal] = useState<boolean>(false);
+
+  return (
+    <>
+      <FieldArray
+        name="owners"
+        render={arrayHelpers => (
+          <>
+            {values.owners.map((owner, index) => (
+              <Row key={`onwer-parent-${index}`} className="py-3">
+                <Container>
+                  <ButtonDiv>
+                    <RemoveButton
+                      onRemove={() => {
+                        setRemoveIndex(index);
+                        setShowRemoveModal(true);
+                      }}
+                    >
+                      Remove Owner
+                    </RemoveButton>
+                  </ButtonDiv>
+                  <H3>Name</H3>
+                  <SectionField label="Given names">
+                    <Input field={`owners[${index}].givenName`} />
+                  </SectionField>
+                  <SectionField label="Last name/Corporation name">
+                    <Input field={`owners[${index}].lastNameOrCorp1`} />
+                  </SectionField>
+                  <SectionField
+                    label="Other name"
+                    tooltip="Additional name for Individual (ex: alias or maiden name or space for long last name) Corporation (ex: Doing Business as) or placeholder for the Last name/Corporate name 2 field from Title"
+                  >
+                    <Input field={`owners[${index}].lastNameOrCorp2`} />
+                  </SectionField>
+                  <SectionField label="Incorporation number">
+                    <Input field={`owners[${index}].incorporationNumber`} />
+                  </SectionField>
+                  <StyledDiv>
+                    <H3>Mailing Address</H3>
+                    <Address namespace={`owners[${index}].address`} />
+                  </StyledDiv>
+                </Container>
+              </Row>
+            ))}
+            <LinkButton
+              data-testid="add-file-owner"
+              onClick={() => {
+                const owner = new AcquisitionOwnerFormModel();
+                arrayHelpers.push(owner);
+              }}
+            >
+              + Add owner
+            </LinkButton>
+
+            <AcquisitionFormModal
+              message="Are you sure you want to remove this row?"
```

Comment:
should this say row or owner?

Reply:
block from another component, just had it replaced with 'Owner'